base/abstractarraymath.jl

```diff
@@ -296,6 +378,21 @@ function repeat(a::AbstractVector, m::Integer)
     return b
 end
 
+function repeat(a::AbstractVector, m::AbstractVector{<:Integer})
```

Comment:
This code doesn't work:
```julia
V1 = [1 2 3 4];
R1= repeat(V1, [3 2 1 1])  # should give [1 1 1 2 2 3 4]
```
giving the error of :
```julia
MethodError: no method matching repelemM(::Array{Int64,2}, ::Array{Int64,2})

You might have used a 2d row vector where a 1d column vector was required.
Note the difference between 1d column vector [1,2,3] and 2d row vector [1 2 3].
You can convert to a column vector with the vec() function.
```
and
```julia
V1 = [1 2 3 4];
R1= repeat(V1, [3, 2, 1, 1])  # should give [1 1 1 2 2 3 4]
```
also gives the error of:
```julia
ArgumentError: m must be a single Integer or an AbstractVector{<:Integer} of same dimension as a
```